src/main/java/Trade/TradePresenter.java

```diff
@@ -0,0 +1,17 @@
+package Trade;
+
+public class TradePresenter implements TradeOutputBoundary{
```

Comment:
Use create() to update the viewmodel. You can find the classes under the ViewModel package.